robocop/checkers/spacing.py

```diff
@@ -517,6 +517,11 @@ def visit_Statement(self, node):  # noqa
 
 
 def get_indent(node):
+    """Calculate the indentation length for given node
+
+    Returns:
+        int: Indentation length
```

Comment:
Maybe we should think about increasing the min required python version of Robocop? That way we could start using typing

Reply:
I was thinking about the same thing exactly when adding these docstrings :D

I would love to have typing, but that would mean that we need to update Python to 3.9 and drop older versions.
I am wondering how this will affect our users.

Reply:
Robotidy uses 3.7 and it already supports typing: https://docs.python.org/3.7/library/typing.html . 3.8 does not introduce anything revolutionary to typing, and 3.9 replaces ``Dict`` or ``List`` types by direct type (``dict``, ``list`` etc). So it would be enough to bump one major version - 3.6 is already at the end of life:

![image](https://user-images.githubusercontent.com/8532066/228124256-674e6f9d-1d4e-4be3-9739-bda7605195a9.png)

Reply:
It's good to actually follow the versions of the dependencies along with their support, and Python version is one of them. I will create another PR for that

Since 3.6 is already dead for over a year, I think we should definitely bump the Python up to 3.7, even for security reasons. 

3.7 is also at the end of its life, so we need to keep an eye on that and update Python versions accordingly. 
I think we can require 3.8 in the second half of the year.